.github/workflows/scheduled-snapshots.yml

```diff
@@ -108,7 +108,7 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.SCHUTZBOT_GH_TOKEN }}
       run: |
         pushd osbuild-composer
-        sed -i -E 's/("baseurl": ".*)-[0-9]{8,12}\/?"/\1-${{ steps.generate_suffix.outputs.suffix }}"/g' Schutzfile
+        perl -pe 's/("baseurl": ".*)(?:(?<!fedora))(?:(?<!fedora-modular))-[0-9]{8,12}\/?"/\1-${{ steps.generate_suffix.outputs.suffix }}"/g' Schutzfile
```

Comment:
I don't fully understand the reason, why would we ever specify fedora repositories?

Reply:
I am proposing a change where even fedora repositories will be pinned in Schutzfile and since `fedora` and `fedora-modular` repository snapshots don't get updated (because they don't change) it will get broken if it's changed by the PR that this action creates.